docs/source/contact.rst

```diff
@@ -5,26 +5,26 @@ Online Forum
 
 Issues
 ----------------
-Questions, feature requests, and bug reports for `MHKiT-Python <https://github.com/MHKiT-Software/MHKiT-Python>`_ and `MHKiT-MATLAB <https://github.com/MHKiT-Software/MHKiT-MATLAB>`_ should be submitted to the GitHub Issues Page. 
+Questions, feature requests, and bug reports for `MHKiT-Python <https://github.com/MHKiT-Software/MHKiT-Python>`_ and `MHKiT-MATLAB <https://github.com/MHKiT-Software/MHKiT-MATLAB>`_ should be submitted to the GitHub Issues Page.
 The GitHub online forums are managed by the MHKiT development team and users.
 
-`Submit MHKiT-Python Issue <https://github.com/MHKiT-Software/MHKiT-Python/issues>`_ 
+`Submit MHKiT-Python Issue <https://github.com/MHKiT-Software/MHKiT-Python/issues>`_
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-`Submit MHKiT-MATLAB Issue <https://github.com/MHKiT-Software/MHKiT-MATLAB/issues>`_ 
+`Submit MHKiT-MATLAB Issue <https://github.com/MHKiT-Software/MHKiT-MATLAB/issues>`_
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Collaboration
 ----------------
-`MHKiT-Python <https://github.com/MHKiT-Software/MHKiT-Python>`_ and `MHKiT-MATLAB <https://github.com/MHKiT-Software/MHKiT-MATLAB>`_ welcomes feedback and code contributions.  
+`MHKiT-Python <https://github.com/MHKiT-Software/MHKiT-Python>`_ and `MHKiT-MATLAB <https://github.com/MHKiT-Software/MHKiT-MATLAB>`_ welcome feedback and code contributions.
 Software developers interested in contributing to the MHKiT open-source software are encouraged to use GitHub to create a `Fork <https://help.github.com/en/github/getting-started-with-github/fork-a-repo>`_ of the repository into their GitHub user account.
-To include your additions to the MHKiT code, please submit a `pull request <https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request>`_ in the master branch of the . 
-Once reviewed by the MHKiT development team, pull requests will be merged into MHKiT master branch, and included in future releases of MHKiT.
-Software developers, within the MHKiT development team and external collaborators, are expected to follow standard practices to document and test new code. 
+To include your additions to the MHKiT code, please submit a `pull request <https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request>`_ in the master branch of the MHKiT GitHub.
```

Comment:
How about ...of the modified repository.

So that we are non-specific but inclusive of either Python or MATLAB